src/admin/components/ResourceTabs/AuditLogTab/components/SiteAuditLogPolygonStatus.tsx

```diff
@@ -71,28 +38,15 @@ const SiteAuditLogPolygonStatus: FC<SiteAuditLogPolygonStatusProps> = ({ record,
         <Text variant="text-14-light" className="mb-4">
           Update the polygon status, view updates, or add comments
         </Text>
-        {recentRequest && (
-          <div className="flex flex-col gap-1 rounded-xl border border-yellow-500 bg-yellow p-4">
-            <div className="flex items-center justify-between">
-              <Text variant="text-16-bold">Change Requested</Text>
-              <Button variant="orange" onClick={deactivateRecentRequest}>
-                Remove Request
-              </Button>
-            </div>
-            <Text variant="text-14-semibold">{recentRequest?.comment}</Text>
-          </div>
-        )}
-      </div>
-      <div className="flex flex-col gap-4">
-        <Text variant="text-16-bold">Polygon Status</Text>
-        <StepProgressbar
-          color="secondary"
-          value={getValueForStatus(record?.meta)}
-          labels={polygonStatusLabels}
-          classNameLabels="min-w-[111px]"
-          className="w-[44%]"
-        />
       </div>
+      <ComentarySection
+        record={{ uuid: record?.uuid, status: record?.meta, title: record?.title }}
+        entity={"SitePolygon"}
```

Comment:
I decided to add the type "Project" | "SitePolygon" to the `entity` property because it provides a safer way to define the type of entity expected in this component.